### DISCUSSION TITLE
app_commands.choices Description on each choice?

### DISCUSSION THREAD

**Discussion by Geuthur** [Rapptz/discord.py#9545] [Q&A]
Hello There,

Is it possible to Add Descriptions to each Choice?

I have this Code

```
    @commands.hybrid_command(name='price', description="Shows the items in the store")
    #@commands.hybrid_command(name='item', description="Shows the items in the store")
    @app_commands.choices(action=[
        app_commands.Choice(name="Mining", value="Mining"), # I want to Add description="Show items from Mining section"
        app_commands.Choice(name="Raid", value="Raid"), # I want to Add description="Show items from Raid section"
        app_commands.Choice(name="Items", value="Items"), # I want to Add description="Show items from Item section"
        ])
    @app_commands.rename(item='item')
    @app_commands.describe(item='Get more information about a specific item')
    async def shop(self, ctx, action: app_commands.Choice[str], item = None):
```
The Example what i want to look
![Screenshot_7](https://github.com/Rapptz/discord.py/assets/761682/1db962fd-c2da-4bce-ad32-f03b64a89791)

The Example what it actually looks

![Screenshot_8](https://github.com/Rapptz/discord.py/assets/761682/e54821a7-d8fa-484b-a6bd-7d7aa961ddc6)

I have tried a lot of thinks but no one works.

Cheers

**Comment by ika2kki** [comment]
sorry, but there's no field for a description or similar on individual choices.
to achieve something of the same effect, you can add some additional text to the `name`, for example:
`Choice(name="Mining (show items from mining section)", value="Mining")`
